### DISCUSSION TITLE
openbmc port 2200 (ssh)

### DISCUSSION THREAD

**Discussion by netllama** [bstansell/conserver#71] [Q&A] [answered]
Has anyone been successful configuring conserver to connect to openBMC's port 2200 instead of the default 623?  

I thought that maybe I could get it to work by simply explicitly specifying the "port" parameter, and it works.  However, when I specified a completely bogus port number, it still worked.  This makes me suspect that conserver is not respecting the port parameter, and continuing to use the default (623).

This is the config that I tried that seemingly ignores the port number:

    console                 c081.mad001.ix  { type ipmi; master     127.0.0.1; host 10.0.0.160; password XXXX ; username QQQ; port 2200; }

Prior to this effort, the config was identical, but did not have the 'port' parameter included.

**Comment by bstansell** [comment, marked as answer]
I took a quick peek at the openbmc docs and it looks like port 2200 is for the ssh protocol, not IPMI.  Conserver doesn't use the port parameter for IPMI as it's a hard-coded thing in the IPMI library.  You'll want to do something a bit different for ssh...a simple example would be:

`console c081.mad001.ix-ssh { type exec; exec ssh -p 2200 10.0.0.160; }`

Then you should be able to attach to the device via either connection type.

**Comment by netllama** [comment]
Thanks, that change does help a lot.  I was able to get it working after some additional tweaks using the following configuration:

    console                 c081.mad001.ix  { type exec; master     127.0.0.1; host 10.0.0.160; exec sshpass -p 'XXX' ssh -o "StrictHostKeyChecking=no" -p 2200 QQQ@10.0.0.160; }
